bundles/aero.minova.rcp.model/src/aero/minova/rcp/model/form/MDetail.java

```diff
@@ -7,6 +7,7 @@
 import java.util.Map;
 
 import org.eclipse.swt.widgets.Control;
+import org.eclipse.ui.forms.widgets.Section;
```

Comment:
Ich glaube es ist nicht gut hier was von der UI einzufügen. Wir haben erst mit Wilfried geredet, dass im MDetail nichts von der UI enthalten sein darf. Kann das noch entfernt werden?

Reply:
Wo ist der unterschied von MDetail und MSection? In der MSection wird das Section Element auch verwendet. Oder soll nur im MDetail kein UI Element verwendet werden?

Reply:
Eigentlich sollte in allen Klassen die mit M beginnen keine UI Elemente enthalten sein.

Reply:
Wir werden die UI Elemente aus den Model Klassen entfernen und über einen Accessor darauf zugreifen. Wir brauchen den ISectionAccessor und den SectionAccessor. Der Sectionaccessor wird dann aufgerufen wenn die UI Elemente abgerufen werden müssen z.B. getSection(). Im ISectionAccessor werden alle Methoden aufgerufen die kein Ui Element zurück liefern.